bin/test_stack

```diff
@@ -79,10 +79,10 @@ sudo ip link set t1sw2-eth28 up
 sleep 50.3231 &
 sleep 32
 test_forch -healthy
-test_varz_value process_state{process=\"faucet\"}
-test_varz_value process_state{process=\"gauge\"}
-test_varz_value process_state{process=\"forch\"}
-test_varz_value process_state{process=\"sleep\"}
+test_varz_value process_state{process=\"faucet\"} forch
+test_varz_value process_state{process=\"gauge\"} forch
+test_varz_value process_state{process=\"forch\"} forch
+test_varz_value process_state{process=\"sleep\"} forch
```

Comment:
make one of this native (empty), and then also add in a negative failing case (e.g. xyzzy)

Reply:
Addressed.